### PR TITLE
Drop web.py from setup.py install_requires and requirements.txt

### PR DESCRIPTION
**Pull request by toabctl**
It's not used anywhere. requirements.txt got updated with
"pip-compile".

### CODE REVIEW

**Comment by toabctl** [comment]
Looks like `teuthology/coverage.py` imports `MySQLdb` which is not mentioned in `setup.py`. I wonder if `teuthology/coverage.py` is still in use?

**Comment by kshtsk** [comment]
@toabctl please close this PR, it will interfere my upcoming ones, hold on on changes setup.py and requirements for awhile.

**Comment by kshtsk** [comment]
There are some tricks with using 'pip-compile' the code will probably have  issues.

File: requirements.txt [hunk 1]
```diff
@@ -55,7 +55,6 @@ markupsafe==1.0           # via jinja2
 monotonic==1.3            # via oslo.utils
 more-itertools==4.3.0     # via pytest
 msgpack-python==0.4.8     # via oslo.serialization
-mysqlclient==1.4.2
```
Comment (kshtsk) [inline requirements.txt]: This is not related change.

Comment (toabctl) [reply requirements.txt]: It's done automatically by `pip-compile`. It's happening because there is no mysql listed in `install_requires` but it's used in `teuthology/coverage.py`.

**Comment by toabctl** [comment]
Closing as suggested in https://github.com/ceph/teuthology/pull/1357#issuecomment-554252412
